cluster-scope/base/operatorgroups/cluster-logging-operator/operatorgroup.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  targetNamespaces:
+    - openshift-logging
```

Comment:
I'm still thinking about this from the #131 perspective. I'm inclined to unblock you here and merge this and then do some global pruning on the namespaced operators in #131. I'm think we need an ADR for operator handling. :smile: 

https://github.com/operate-first/blueprint/issues/23

@HumairAK mind you take a look at this PR as well?